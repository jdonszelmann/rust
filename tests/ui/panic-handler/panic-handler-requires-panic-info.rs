//@ compile-flags:-C panic=abort

#![feature(lang_items)]
#![feature(no_core)]
#![no_core]
#![no_main]

#[panic_handler]
//~^ ERROR cannot find attribute `panic_handler` in this scope
fn panic() -> ! {
    loop {}
}

#[lang = "sized"]
trait Sized {}
