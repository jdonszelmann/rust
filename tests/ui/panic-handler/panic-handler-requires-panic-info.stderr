error: cannot find attribute `panic_handler` in this scope
  --> $DIR/panic-handler-requires-panic-info.rs:8:3
   |
LL | #[panic_handler]
   |   ^^^^^^^^^^^^^

error: aborting due to 1 previous error

